src/components/recommendation-list/recommendation-list.js

```diff
@@ -2,34 +2,38 @@ import { useState } from 'react'
 import ReactTooltip from 'react-tooltip'
 import PropTypes from 'prop-types'
 import './recommendation-list.scss'
-import AssignmentOutlinedIcon from '@material-ui/icons/AssignmentOutlined'
 import { CopyToClipboard } from 'react-copy-to-clipboard'
 import { RecommendationUtils } from '../../utils'
-
+import FilterNoneIcon from '@material-ui/icons/FilterNone'
 
 const RecommendationList = props => {
   const { recommendations } = props
 
   const [showTooltipAsCopied, setShowTooltipAsCopied] = useState(false)
   const createHoverText = (systemName) => showTooltipAsCopied ? 'Copied!' : `Copy table name '${systemName}'`
 
+  const onCopyClick = () => {
+    setShowTooltipAsCopied(true)
+  }
+
   const recommendationItems = () => {
     return recommendations.map(rec => {
```

Comment:
Are you using a 80 character width monitor over there? It's not the end of the world, but it would be nice to call this `recommendation` instead of `rec` - Ben